### ISSUE TITLE
Request: prevent (or grey-out) blocks being dragged into stage scripting area

### ISSUE THREAD

**Issue by davelai** [jmoenig/Snap--Build-Your-Own-Blocks#962]
The "Stage" scripts exclude motion blocks. However, you are able to drag these blocks into the Stage scripting area from another sprite. I propose making these blocks either grey-out if dragged in, or prevent them from being dragged in altogether.  Thanks!

**Comment by jmoenig** [comment]
Good point. Currently the only way to drag blocks out of context is by "cheating" them out of an open script editor. If someone does that they can produce strange results. However, in the next version (4.1) you'll be able to drag any block anywhere, so I'm reluctant to disable it now (because I'll have to re-enable it soon).

**Comment by davelai** [comment]
Sounds good! This isn't urgent... just a suggestion.

**Comment by cycomachead** [comment]
Going through these issues...

Jens: You don't even need to "cheat" blocks by using a script editor, you can simply drag them from one sprite like you might normally duplicate blocks. 

However, IMO, having motion blocks on the stage is _currently_ useful, as in the case of using `tell` to run them in the context of another sprite.
